src/views/htmlcontent/src/js/components/app.component.ts

```diff
@@ -40,25 +40,25 @@ export interface IGridDataSet {
 
 // tslint:disable:max-line-length
 const template = `
-<div class="fullsize vertBox">
-    <div *ngIf="dataSets.length > 0" role="button" id="resultspane" tabIndex="0" class="boxRow header collapsible"
+<div class="fullsize vertBox" tabindex="-1">
```

Comment:
-1 means not keyboard focusable, but programmatically focusable. why do we need to set it as -1 here?

Reply:
The person specifically requested the tab key to not go to the whole section: https://github.com/microsoft/vscode-mssql/issues/17209 @alanrenmsft, in my findings I found that this was the window for the whole element, I decided to get rid of the keyboard focus because it would allow the user to tab to the slickgrid table directly. (Is there a way to make the entire element itself not focusable at all?)

Reply:
a div is by default not keyboard focusable